can_messages.asciipb

```diff
@@ -127,7 +127,7 @@ msg {
 # IDs: 9-14 Reserved
 msg {
   id: 15
-  source: DRIVER_CONTROLS
+  source: DRIVER_CONTROLS_CENTER_CONSOLE
```

Comment:
Maybe it makes sense to add a Signal denoting which of the motor controllers to reset?

There probably isn't a good reason (anymore) to reset both at the same time.

Reply:
I removed the motor controller reset message for now since it doesn’t do anything just to get things working but I plan on implementing it in the future.